app/main.py

```diff
@@ -2,0 +1,35 @@
+class CommandNameError(Exception):
+    """CommandNameError"""
+
+
+class DoubleNameError(Exception):
+    """DoubleNameError"""
+
+
+class LanCommandError(Exception):
+    """LaneCommandError"""
+
+
+def copy_file(command: str) -> None:
+    command_elements = command.split()
+
+    if len(command_elements) < 3:
+        raise LanCommandError(
+            "it must start with 'cp' and include two file names"
+            " 'cp input_file_name.txt output_file_name.txt'"
+        )
+
+    if command_elements[0] != "cp":
+        raise CommandNameError(f"'{command_elements[0]}' is not 'cp' command")
```

Comment:
Bug: The 'cp' command is not being checked correctly. The code should check if the first element in the `command_elements` list is 'cp', not if it is not 'cp'. This could lead to incorrect behavior if the command is not 'cp'.